packages/utilities/package.json

```diff
@@ -262,9 +262,19 @@
 			"require": "./dist/lib/toTitleCase.js"
 		},
 		"./tryParse": {
```

Comment:
Should probably remove this

Reply:
it'll be a breaking change ig?

Reply:
Indeed, removing functions is a breaking change